.github/workflows/run-bdd-tests.yml

```diff
@@ -21,8 +21,9 @@ jobs:
       - name: Build BDD covered packages and dependencies
         run: |
           yarn exec turbo -- run build --filter="./packages/manager/core/*" --concurrency=5
-          yarn exec turbo -- run build --filter="./packages/manager-react-components" --concurrency=5
+          yarn exec turbo -- run build --filter="./packages/manager-react-components" 
           yarn exec turbo -- run build --filter="./packages/manager/modules/order" --concurrency=5
+          yarn exec turbo -- run build --filter="./packages/manager/modules/manager-pci-common" 
```

Comment:
Can you also remove `--concurrency=5` in Line 25?

Reply:
Sure